Index.html

```diff
@@ -0,0 +1,343 @@
+              <a href="#" target="_blank">
+                <img src="images/github.png" alt="git" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/Linkedin.png" alt="linkedIn" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/angelite.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/twitter.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/medium.png" alt="image" />
+              </a>
+            </li>
+          </ul>
+          <div class="new-code">
+            <button type="submit" class="go-newmes">My resume</button>
+          </div>
+          <ul class="skill">
+            <li class="event12">
+              <li class="event7">
+                <h3 class="languages">Languages</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+                </button>
+              </li>
+              
+                <ul class="language-list">
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
+                    </div>
+                  </li>
+                </ul>
+              
+            </li>
+            <ul class="event-skill">
+              <li class="event18">
+                <h3>Frameworks</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+              <li class="event18">
+                <h3>Skills</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="consection">
+      <ul class="contain">
+        <li class="contact">
+          <h1 >Contact me</h1>
+          <p>
```

Comment:
- [ ] You have done a great job👍 of implementing changes suggested by the previous reviewer🤓. However, this paragraph is still not aligned properly. 🙏Please remove the `</br>` tag and align this paragraph with `text-align: center` to match the Figma design. This will enhance the appearance of your design and make it as cool as the Figma design.

Reply:
Hi @abdallahmalima changes were made successfully.

</br> removed

![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/9d6b5131-b1d4-489c-ad87-4a245b224b72)

text-align: center were applied.

![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/137fee8d-8114-4425-b5d3-48cd669e31f8)